src/SingleBookingPage/SelectedBooking.js

```diff
@@ -23,15 +22,8 @@ type RenderState = {
 
 const selectedBookingQuery = graphql`
   query SelectedBookingQuery($id: ID!) {
-    booking(id: $id) {
-      type
-      oneWay {
-        ...BookingDetail_booking
-      }
-      return {
-        ...BookingDetail_booking
-      }
-      multicity {
+    node(id: $id) {
```

Comment:
Huh, not this query, at least now, I should have clarified it before probably more clearly - we need to keep the old `booking` because we need to support old booking ids from frontend

Reply:
Old booking id === number, GraphQL id === opaque string

Reply:
The problem is when I refactored `BookingCardsList` `SelectedBooking` stoped working and I dunno why, because `id`s are the same 😐.. If you could checkout to `3b5fbe47163fcf651b9549389e8e248a07ac6baf` and try it that would be great!

Reply:
`booking(id: ID)` query accept both opaque string IDs and numerical `databaseId`, but `node` only string ID and we need both because frontend use only numerical

Therefore let's stick with `booking` query